src/fragment-components/a-radiobutton.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { RadioButton, TextInput, Checkbox } from 'carbon-components-react';
+import { AComponent, ComponentInfo } from './a-component';
+import { ComponentCssClassSelector } from '../components/css-class-selector';
+import { useFragment } from '../context';
+import { css } from 'emotion';
+import { getParentComponent, updatedState, Adder } from '../components';
+import image from './../assets/component-icons/radiobutton.svg';
+import { nameStringToVariableString,
+		angularClassNamesFromComponentObj,
+		reactClassNamesFromComponentObj } from '../utils/fragment-tools';
+
+export const ARadioButtonStyleUI = ({selectedComponent, setComponent}: any) => {
+	return <>
+		<Checkbox
+			labelText='Disable button'
+			id='disable'
+			checked={selectedComponent.disabled}
+			onChange={(checked: boolean) => setComponent({
+				...selectedComponent,
+				disabled: checked
+			})}/>
+		<TextInput
+			value={selectedComponent.labelText || ''}
+			labelText='Radio button label'
+			placeholder='Button value'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					labelText: event.currentTarget.value
+				});
+			}}
+		/>
+		<ComponentCssClassSelector componentObj={selectedComponent} setComponent={setComponent}/>
+	</>
+};
+
+const addButtonCss = css`
+	position: relative;
+`;
+
+export const ARadioButton = ({
+	children,
+	componentObj,
+	selected,
+	...rest
+}: any) => {
+	const [fragment, setFragment] = useFragment();
+	const parentComponent = getParentComponent(fragment.data, componentObj);
+
+	const addRadio = (offset = 0) => setFragment({
+		...fragment,
+		data: updatedState(
+			fragment.data,
+			{
+				type: 'insert',
+				component: {
+					type: 'radioButton',
+					value: `${componentObj.id}`,
+					labelText: 'New Option',
+					disabled: false
+				}
+			},
+			parentComponent.id,
+			parentComponent.items.indexOf(componentObj) + offset
+		)
+	});
+	componentObj.value = `${componentObj.id}`;
+	return	(
+		<Adder
+			active={selected}
+			addButtonsCss={addButtonCss}
+			key={componentObj.id}
+			bottomAction={() => addRadio(1)}>
+			<AComponent
+				selected={selected}
+				headingCss={css`width: fit-content; min-width: 9rem;`}
+				componentObj={componentObj}
+				{...rest}> 
+					<RadioButton
+						id={componentObj.id}
+						name={componentObj.codeContext?.name}
+						labelText={componentObj.labelText}
+						defaultChecked={componentObj.defaultChecked}
+						checked={componentObj.defaultChecked}
+						value= {componentObj.value}
+						disabled= {componentObj.disabled}/>
+			</AComponent>
+		</Adder>
+	)
```

Comment:
Using a tab after return, missing semicolon, and props should be aligned in the following way:
```suggestion

	return (
		<Adder
		active={selected}
		addButtonsCss={addButtonCss}
		key={componentObj.id}
		bottomAction={() => addRadio(1)}>
			<AComponent
			selected={selected}
			headingCss={css`width: fit-content; min-width: 9rem;`}
			componentObj={componentObj}
			{...rest}> 
				<RadioButton
					id={componentObj.id}
					name={componentObj.codeContext?.name}
					labelText={componentObj.labelText}
					defaultChecked={componentObj.defaultChecked}
					checked={componentObj.defaultChecked}
					value= {componentObj.value}
					disabled= {componentObj.disabled} />
			</AComponent>
		</Adder>
	);
```

Reply:
dont know why it shows up like this on github..